css/style.css

```diff
@@ -89,7 +90,7 @@ a.helpfulLink {color:#526972;}
 .smallDarkRedText{font-size:85%;color:#7a0026;}
 .normalText{color:#3D545D}
 .smallBlueText{font-size:85%;font-weight:normal; color:#526972}

```

Comment:
be consistent : if you change smallGreyText to have a final ";", change also the smallBlueText, the normalText